### PR TITLE
feat: [SRM-11183]: fixing scroll issues for multiple screens

### PR DESCRIPTION
**Pull request by harshilbits**
#### Screenshots
Before:
New Note
New Note
New Note

After:
New Note

New Note

New Note

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by harshilbits** [comment]
retrigger titlecheck

**Comment by harshilbits** [comment]
retrigger titlecheck

File: src/modules/85-cv/pages/monitored-service/MonitoredServicePage.module.scss [hunk 1]
```diff
@@ -26,4 +26,5 @@
 
 .pageBody {
   max-height: calc(100vh - 140px);
+  overflow-y: scroll !important;
```
Comment (abhinavrastogi-harness) [inline src/modules/85-cv/pages/monitored-service/MonitoredServicePage.module.scss:29]: pls see https://harness.slack.com/archives/C035UJ8DA6Q/p1658208059163809
the right way to fix this is to set `min-height: var(--page-min-height)`.
remove max-height and overflow-y.

**Review by abhinavrastogi-harness — CHANGES_REQUESTED** [review]

File: src/modules/85-cv/pages/monitored-service/MonitoredServicePage.module.scss (hunk 1 above)
Comment (abhinavrastogi-harness) [reply src/modules/85-cv/pages/monitored-service/MonitoredServicePage.module.scss:29]: infact, you should not override pagebody at all, and everything will work correctly

Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/MonitoredServicePage.module.scss:29]: sure @abhinavrastogi-harness

Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/MonitoredServicePage.module.scss]: done.

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Comment by harshilbits** [comment]
Coverage is not meeting in the existing code.

**Pull request merged**